### DISCUSSION TITLE
IOS HEIC support

### DISCUSSION THREAD

**Discussion by alainbuysse** [directus/directus#12805] [Feature Requests]
Till now the IMG markup in HTML5 does not support IOS HEIC format.
Would be nice to have an autoconvert to jpeg or custom format when uploading HEIC-files?

Edit: just found it: we can change the format on our mobiledevice from heic, to jpg

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
